lib/main_common.pm

```diff
@@ -848,4 +849,15 @@ sub load_syscontainer_tests() {
     loadtest 'virtualization/syscontainer_image_test';
 }
 
+sub load_sles4sap_tests {
```

Comment:
`lib/main_common.pm` is only meant for tests which are "common" among multiple distributions, e.g. "openSUSE" *and* "SLE" but "sles4sap" certainly does not count as that so please move that to products/sle/main.pm

Reply:
Understood.

Reply:
Done